airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -433,33 +469,35 @@ private List<UUID> createOperations(final WebBackendConnectionCreate webBackendC
     return operationIds;
   }
 
-  private List<UUID> updateOperations(final WebBackendConnectionUpdate webBackendConnectionUpdate)
+  private List<UUID> updateOperations(final ConnectionRead connectionRead, final WebBackendConnectionUpdate webBackendConnectionPatch)
```

Comment:
can we call this `createAndUpdateOperations` to make clear this is also creating?